hacker/admin.py

```diff
@@ -1,14 +1,39 @@
 from django.contrib import admin
 from .models import Hacker, Application, Confirmation, Team
 from django import forms
+import datetime
 
 
+''' `Hacker` '''
+def make_checked_in(modeladmin, request, queryset): # Needs to be Tested!!!
+    queryset.update(checked_in=True)
+    queryset.update(checked_in_datetime=datetime.datetime.now())
+
 class HackerAdmin(admin.ModelAdmin):
+    list_display = ('username','email','first_name', 'last_name', 'is_active', 'is_staff', 'checked_in')
     fieldsets = [
         ('User Information', {'fields': ['first_name','last_name','email','username','password']}),
         ('Advanced',         {'fields': ['is_superuser','is_staff','is_active'], 'classes': ['collapse']}),
     ]
 
+    make_checked_in.short_description = "Check-In Selected Hackers"
+    actions = [make_checked_in]
+
+    def has_add_permission(self, request, obj=None):
+        return True
+
+    def has_change_permission(self, request, obj=None):
+        return False
+
+#    def has_delete_permission(self, request, obj=None):
+#        return False
+    
+''' `Application` '''
+def make_approved(modeladmin, request, queryset): # Needs to be Tested!!!
```

Comment:
See above comment regarding `make_` functions.
`approve` is a good alternative.